Assets/Source/Components/Player/PlayerBehavior.cs

```diff
@@ -59,17 +73,41 @@ public override void ComponentAwake()
 
             cameraEffector = GetRequiredComponent<CameraEffectComponent>(cameraObject);
             menuSelector = GetRequiredComponent<CanvasMenuSelectorComponent>(FindOrCreateCanvas());
-            levelDirector = GetRequiredComponent<DirectorComponent>(FindLevelObject());
 
             actorDash = gameObject.AddComponent<ActorDash>();
             actorDash.CooldownTime = DASH_COOLDOWN;
             actorDash.DashDistance = DASH_DISTANCE;
 
             blasterSound = GetRequiredResource<AudioClip>($"{ResourcePaths.SoundFXFolder}/Player/playerBlaster");
 
+            GameObject hud = GetRequiredChild("HUD", FindOrCreateCanvas());
+
+            GameObject ammoDisplay = GetRequiredChild("AmmoDisplay", hud);
+            GameObject ammoPlaceholder = GetRequiredChild("AmmoPlaceholder", ammoDisplay);
```

Comment:
I don't think we should call this GameObject "AmmoPlaceholder" because that makes me think it'll be deleted eventually

Reply:
renamed